content/fr/integrations/amazon_ecs.md

```diff
@@ -292,44 +292,37 @@ Pour réduire les données parasites, l'intégration AWS ECS est automatiquemen
 
 {{< img src="integrations/amazon_ecs/aws_ecs_events.png" alt="Événements AWS ECS" responsive="true">}}
 
-Pour supprimer ce filtre et recevoir tous les événements de votre intégration Datadog/AWS ECS, contactez [l'assistance Datadog][15].
+Pour supprimer ce filtre et recevoir tous les événements de votre intégration Datadog/AWS ECS, contactez [l'assistance Datadog][19].
 
 ### Checks de service
 L'intégration AWS ECS n'inclut aucun check de service.
 
 ## Dépannage
-Besoin d'aide ? Contactez [l'assistance Datadog][15].
+Besoin d'aide ? Contactez [l'assistance Datadog][19].
 
 ## Pour aller plus loin
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: https://docs.datadoghq.com/fr/agent/
+
+
+{{< get-dependencies >}}
+[1]: https://docs.datadoghq.com/fr/agent
 [2]: https://docs.datadoghq.com/fr/integrations/faq/agent-5-amazon-ecs
 [3]: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_GetStarted_EC2.html
-[4]: https://aws.amazon.com/cli/
-[5]: /fr/json/datadog-agent-ecs.json
-[6]: https://app.datadoghq.com/account/settings#api
-[7]: http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html#ecs-config-s3
-[8]: https://docs.datadoghq.com/fr/integrations/amazon_web_services/#installation
-[9]: https://docs.aws.amazon.com/IAM/latest/UserGuide/list_ecs.html
-[10]: http://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_container_instance.html
-[11]: https://docs.datadoghq.com/fr/logs/docker/
-[12]: http://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html#w2ab1c21c21c13
-[13]: https://docs.datadoghq.com/fr/integrations/amazon_web_services/#create-a-new-lambda-function
-[14]: https://github.com/DataDog/dogweb/blob/prod/integration/amazon_ecs/amazon_ecs_metadata.csv
-[15]: https://docs.datadoghq.com/fr/help/
-[18]: https://docs.datadoghq.com/fr/logs/log_collection/docker/?tab=containerinstallation#activate-log-integrations
-[19]: /fr/agent/faq/agent-commands/#start-stop-restart-the-agent
-[20]: https://docs.datadoghq.com/fr/logs/processing/#log-processing
-[21]: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html#service_scheduler_daemon
-[22]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
-[23]: https://docs.datadoghq.com/fr/agent/autodiscovery/
-[24]: https://docs.datadoghq.com/fr/tracing/setup/docker/
-[25]: /fr/json/datadog-agent-ecs1.json
-[26]: https://github.com/DataDog/datadog-trace-agent
-[27]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
-[28]: https://docs.datadoghq.com/fr/tracing/advanced_usage/?tab=java#change-agent-hostname
-
-
-{{< get-dependencies >}}
\ No newline at end of file
+[4]: https://docs.datadoghq.com/fr/tracing/setup/docker
+[5]: https://aws.amazon.com/cli
+[6]: /json/datadog-agent-ecs.json
+[7]: /json/datadog-agent-ecs1.json
+[8]: https://app.datadoghq.com/account/settings#api
+[9]: #collecte-de-processus
```

Comment:
Since this is on the same page, shouldn't it be linked above?

Reply:
we could but it's not mandatory since it's a translated files and it might be override soon